ee/models/session_recording_extensions.py

```diff
@@ -65,41 +63,62 @@ def persist_recording(recording_id: str, team_id: int) -> None:
         recording.save()
         return
 
-    recording.load_snapshots(100_000)  # TODO: Paginate rather than hardcode a limit
-    analytics_payload["snapshots_load_time_ms"] = (
-        timezone.now() - start_time
-    ).total_seconds() * 1000 - analytics_payload["metadata_load_time_ms"]
-
-    content: PersistedRecordingV1 = {
-        "version": "2022-12-22",
-        "distinct_id": recording.distinct_id,
-        "snapshot_data_by_window_id": recording.snapshot_data_by_window_id,
-    }
-
-    # TODO: This is a hack workaround for datetime conversion
-    string_content = json.dumps(content, default=str)
-    analytics_payload["content_size_in_bytes"] = len(string_content.encode("utf-8"))
-    string_content = compress_to_string(string_content)
-    analytics_payload["compressed_size_in_bytes"] = len(string_content.encode("utf-8"))
-
-    logger.info("Persisting recording: writing to S3...", recording_id=recording_id, team_id=team_id)
-
-    try:
-        object_path = recording.build_object_storage_path()
-        object_storage.write(object_path, string_content.encode("utf-8"))
-        recording.object_storage_path = object_path
-        recording.save()
-
-        analytics_payload["total_time_ms"] = (timezone.now() - start_time).total_seconds() * 1000
-        report_team_action(recording.team, "session recording persisted", analytics_payload)
+    # if snapshots are already in blob storage, then we can just copy the files between buckets
+    with SNAPSHOT_PERSIST_TIME_HISTOGRAM.labels(source="S3").time():
+        target_prefix = recording.build_object_storage_path()
```

Comment:
Why don't we use the same format that we have for the non-lts files? I know its different to the previously stored data but feels worth the switch. We could even manually migrate the old files over.